src/v/datalake/batching_parquet_writer.cc

```diff
@@ -43,12 +49,24 @@ batching_parquet_writer::initialize(std::filesystem::path output_file_path) {
           ss::open_flags::create | ss::open_flags::truncate
             | ss::open_flags::wo);
     } catch (...) {
+        vlogl(
+          datalake_log,
+          ss::log_level::error,
```

Comment:
use `vlog(datalake_log.error)` -- `vlogl` is only useful when you want to dynamically select the log level.

Reply:
Fixed